### PR TITLE
improve User Guide icon

### PR DESCRIPTION
**Pull request by michaelchadwick**
Fixes ilios/ilios#5609

My attempt to address all the issues:
* Color scheme made consistent
* Icon shape made consistent
* Question mark uses the same base font as all other Fontawesome icons, so...maybe not fixable?
* Large screen icon alignment fixed

To do this, I had to go from using an `<a>` (which I chose since the icon takes you somewhere instead of just doing something) to using a `<button>` (which all the other elements use, giving it a lot of the same existing CSS properties) and adding a component class to model the "click this to open a new window/tab" action

### CODE REVIEW

**Review by jrjohnson — COMMENTED** [review]
I agree this should be a link. I'd rather keep the semantically correct HTML, but if that isn't possible I'll accept it. Take a look at the `ilios-link-button` mixin though, it may help.

**Comment by michaelchadwick** [comment]
I'll take another crack at this to keep the semantic `<a>`-ness of it, but I wanted to get it out there for others to look at for visual and functional reasons.

**Comment by michaelchadwick** [comment]
Changed back from `<button>` to `<a>` so semantic again. Had to fudge some CSS here and there, but I think it is both a bit smaller and a more distant from the other icons.

**Review by jrjohnson — APPROVED** [review]
Excellent. Looks great to me.

**Comment by saschaben** [comment]
please if we are able: align center vertically, rather than align-bottom viz the other icons; add 2px padding on right, since when vertical scroll appears, it overlays the right edge of the icon.

otherwise, all A-OK

**Comment by michaelchadwick** [comment]
@saschaben Thanks for the feedback. Will hammer it out some more soon.

**Comment by michaelchadwick** [comment]
@saschaben Made some adjustments. Let me know if this works better!

File: packages/frontend/app/styles/components/ilios-header.scss [hunk 1]
```diff
@@ -38,7 +38,11 @@
     }
     .user-guide-link {
       grid-area: guide;
-      height: 100%;
+      height: 72%;
```
Comment (jrjohnson) [inline packages/frontend/app/styles/components/ilios-header.scss:41]: Is this a magic number that looks good, or a calculation based on something else? I'm ok with either, but just wanted to understand where it was coming from. Might need a comment to that effect.

Comment (michaelchadwick) [reply packages/frontend/app/styles/components/ilios-header.scss]: I switched this out for a flexbox implementation instead. Thanks for the nudge ;P

**Review by jrjohnson — APPROVED** [review]
Excellent

**Pull request merged**